projects/demo/src/modules/components/island/island.template.html

```diff
@@ -40,12 +40,12 @@
                 <ul class="tui-list tui-list_linear tui-list_nested tui-list_extra-small">
                     <li class="tui-list__item">
                         <code>.tui-island__paragraph_link</code>
-                        : parahraph with links. Margin top
+                        : paragraph with links. Margin top
                         <code>16px</code>
                     </li>
                     <li class="tui-list__item">
                         <code>.tui-island__paragraph_button</code>
-                        : parahraph with buttons. Margin to
+                        : paragraph with buttons. Margin to
```

Comment:
how did you find that typo :D

Reply:
I noticed it before (when I was refactoring `Action`). Then I forgot about it.
And now (when I need to trigger changes inside `demo` somehow) I've recalled about it ))